groups/bsl/bslstl/bslstl_variant.h

```diff
@@ -0,0 +1,3338 @@
+    variant(bsl::allocator_arg_t, allocator_type allocator);
+        // Create variant object containing the 0th alternative.  If variant is
+        // allocator aware, use the specified allocator to supply memory.
+
+    variant(bsl::allocator_arg_t, allocator_type allocator, const variant&);
+    variant(bsl::allocator_arg_t,
+            allocator_type                          allocator,
+            BloombergLP::bslmf::MovableRef<variant>);
+        // Create variant object containing the same alternative as the
+        // 'original' object.  If 'variant' is allocator aware, use the
+        // specified allocator to supply memory.  If the 'original' object is
+        // not 'valueless_by_exception', the contained alternative object is
+        // created by copy construction.  The current copy/move construction
+        // implementation is not 'constexpr'.
+
+    template <class TYPE>
+    variant(bsl::allocator_arg_t,
+            allocator_type                                           allocator,
+            BSLS_COMPILERFEATURES_FORWARD_REF(TYPE),
+            BSLSTL_VARIANT_DECLARE_IF_CONSTRUCTS_FROM(variant, TYPE));
+
+    template <class TYPE>
+    explicit variant(bsl::allocator_arg_t,
+                     allocator_type             allocator,
+                     bsl::in_place_type_t<TYPE>);
+    template <class TYPE, class ARG>
+    explicit variant(bsl::allocator_arg_t,
+                     allocator_type                         allocator,
+                     bsl::in_place_type_t<TYPE>,
+                     BSLS_COMPILERFEATURES_FORWARD_REF(ARG) arg);
+        // Create variant object with an alternative 'TYPE' created in place
+        // using the specified 'arg'.  If variant is allocator aware, use the
+        // specified allocator to supply memory.
+
+    template <size_t INDEX>
+    explicit variant(bsl::allocator_arg_t,
+                     allocator_type               allocator,
+                     bsl::in_place_index_t<INDEX>);
+    template <size_t INDEX, class ARG>
+    explicit variant(bsl::allocator_arg_t,
+                     allocator_type                         allocator,
+                     bsl::in_place_index_t<INDEX>,
+                     BSLS_COMPILERFEATURES_FORWARD_REF(ARG) arg);
+        // Create variant object with an alternative with index 'INDEX' created
+        // in place using the specified 'arg'.  If variant is allocator aware,
+        // use the specified allocator to supply memory.
+
+#endif  //BSL_VARIANT_FULL_IMPLEMENTATION
+
+    // 20.7.3.2, destructor
+    ~variant();
+        // Destroy the managed alternative object, if it exists.
+
+    // MANIPULATORS
+    // 20.7.3.4, modifiers
+#ifdef BSL_VARIANT_FULL_IMPLEMENTATION
+    template <class TYPE, class... ARGS>
+    TYPE& emplace(ARGS&&...);
+        // Create an object of alternative 'TYPE' created in place using the
+        // specified 'args' and return a reference providing modifiable access
```

Comment:
I think it suffices to simply say "return a reference to the underlying 'TYPE' object". The fact that the user is allowed to write through it is implied (if it were not so, we would choose to make it `const` to prevent this).

Reply:
done. Also applied to all other instances of the same phrase.